auth/src/main/java/com/firebase/ui/auth/ui/account_link/SaveCredentialsActivity.java

```diff
@@ -60,11 +60,19 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.save_credentials_layout);
+
+        // If SmartLock is not enabled, finish immediately
+        if (!getIntent().getBooleanExtra(ExtraConstants.EXTRA_SMARTLOCK_ENABLED, true)) {
```

Comment:
Ah I see the activity is used to start resolutions ... OK I have to think of something else